app/controllers/api/v1/my_requests_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Api
+  module V1
+    class MyRequestsController < ApplicationController
+      before_action :set_headers
+
+      def index
+        my_requests = TripJoinRequest.where(user_id: current_user.id)
```

Comment:
The `where` can't be changed to `find` because it searches trip join requests by its **user id** and not by its **id**. Instead, I will change it to `find_by_user_id`.